extensions/blocks/podcast-player/style.scss

```diff
@@ -119,9 +119,20 @@ $player-float-background: $light-gray-200;
 	}
 
 	.jetpack-podcast-player__audio-player {
+		height: 40px; /* the same as .mejs-container */
```

Comment:
Should we use a variable for this `40px` value to document it at the top of the file?